<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BikramKesharis-Mac-mini.local" failures="1" tests="7" name="Test" time="78.138" errors="0" timestamp="2024-06-28T10:43:31 IST">
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="openAppointment" time="9.903"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="fstDoctorAppointment" time="16.435"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="closeAppointment" time="3.598"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="fstDoctorLogin" time="22.029"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="assignToDoctor" time="1.153">
    <failure type="java.lang.AssertionError" message="expected [Assigned successfully] but found [Updated Successfully]">
      <![CDATA[java.lang.AssertionError: expected [Assigned successfully] but found [Updated Successfully]
at com.akhilsystem_release.test.emr.DoctorAssign.assignToDoctor(DoctorAssign.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.akhilsystem_release.test.emr.TestRunner.main(TestRunner.java:49)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- assignToDoctor -->
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="assignedDocLogin" time="11.74"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="verifySndDoctorData" time="4.763"/>
</testsuite> <!-- Test -->
